apps/prairielearn/src/cron/sendExternalGraderStats.ts

```diff
@@ -7,7 +9,24 @@ import * as opsbot from '../lib/opsbot.js';
 export async function run() {
   if (!opsbot.canSendMessages()) return;
 
-  const result = await sqldb.callOneRowAsync('grading_jobs_stats_day', []);
+  const result = await sqldb.callRow(
+    'grading_jobs_stats_day',
+    z.object({
+      count: z.bigint(), // Is this good?
```

Comment:
In many of these cases we're casting the count to int to avoid this problem (e.g., https://github.com/PrairieLearn/PrairieLearn/blob/2e9db6d288ff1b49d290ae441755a33c04c660e0/apps/prairielearn/src/lib/question-testing.sql#L3), on the assumption that we'll likely have less than 4 billion entries. This avoids the problem with having to handle bigint.